dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -240,16 +242,18 @@ public Response getContentTypeFieldByVar(@PathParam("typeId") final String typeI
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Produces({ MediaType.APPLICATION_JSON, "application/javascript" })
 	public Response updateContentTypeFieldById(@PathParam("typeId") final String typeId, @PathParam("fieldId") final String fieldId, 
-			final String fieldJson, @Context final HttpServletRequest req) throws DotDataException, DotSecurityException {
+			final String fieldJson, @Context final HttpServletRequest req) {
 
 		final InitDataObject initData = this.webResource.init(null, false, req, false, null);
 		final User user = initData.getUser();
 		final FieldAPI fapi = APILocator.getContentTypeFieldAPI();
 
-		Response response = null;
+		Response response;
 	
 		try {
 			Field field = new JsonFieldTransformer(fieldJson).from();
+			field = FieldBuilder.builder(field).contentTypeId(typeId).build();
```

Comment:
I think we should set this to `ContentTypeAPI.find(typeId).id()` - this would allow someone to use either the UUID or the variable name in the call.

Reply:
This was a good catch, btw.